src/main/java/org/pih/petl/job/type/SqlServerImportJob.java

```diff
@@ -84,17 +84,17 @@ public void execute(final ExecutionContext context) throws Exception {
         String targetTable = config.getString("load", "table");
 
         // Get target table schema
+        String targetSchema = null;
         String targetSchemaFilename = config.getString("load", "schema");
-        ConfigFile targetSchemaFile = appConfig.getConfigFile(targetSchemaFilename);
-        String targetSchema = targetSchemaFile.getContents();
-
-        // TODO: Add validation in
+        if (StringUtils.isNotEmpty(targetSchemaFilename)) {
```

Comment:
This just allows the schema to be optional.  It may be that we don't want the job to take responsibility over creating the schema, but may want that to happen explicitly via an earlier job.